services/resource-usage-tracker/docker-compose-extra.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Includes all adjancent services for testing/development purposes
+#
+version: "3.8"
+services:
+  postgres:
+    image: postgres:14.5-alpine@sha256:db802f226b620fc0b8adbeca7859eb203c8d3c9ce5d84870fadee05dea8f50ce
```

Comment:
This image is not the one currently used in simcore (master), see : https://github.com/ITISFoundation/osparc-simcore/pull/4368

Reply:
Ok thank you, this is there just to be used for local development if necessary, but to be honest, didn't used it in the end -> I can remove it, there are already unit tests with DB.